src/python/pants/option/global_options.py

```diff
@@ -96,12 +96,11 @@ def register_bootstrap_options(cls, register):
                   'subprocesses that outlive the workdir data (e.g. `./pants server`).')
     register('--pants-config-files', advanced=True, type=list,
```

Comment:
The length of this option name is bugging me a bit, given that we've in the past recommended `--config-override` as a quick and easy way to toggle between execution modes.

Could we register a tiny alias for this option and recommend that?
```
register('-c', '--pants-config-files', advanced=True, type=list, ...)
```